tests/sensors/test_base.py

```diff
@@ -44,6 +42,8 @@
 from airflow.models import TaskReschedule
 from airflow.models.xcom import XCom
 from airflow.operators.empty import EmptyOperator
+from airflow.providers.celery.executors.celery_executor import CeleryExecutor
+from airflow.providers.celery.executors.celery_kubernetes_executor import CeleryKubernetesExecutor
```

Comment:
Not in core, but we still do in tests. I do not **yet** want to disentangle those - because some of our tests in Airflow actually use either Celery or Kubernetes executor. Those are not necessary to be removed from core **just yet**. We will deal with it later if/when we actually split providers more.